lib/rewrite/rewrite-subst.c

```diff
@@ -50,16 +50,19 @@ void
 log_rewrite_subst_process(LogRewrite *s, LogMessage **pmsg, const LogPathOptions *path_options)
 {
   LogRewriteSubst *self = (LogRewriteSubst *) s;
+  LogMessage *msg;
+  NVTable *nvtable;
```

Comment:
Do we want to keep the early declaration scheme in new or refactored code?
This is not a request for change, I'm just asking it in general.